frontend/packages/dev-console/src/components/topology/Topology.tsx

```diff
@@ -51,6 +52,7 @@ interface StateProps {
   filters: TopologyFilters;
   application: string;
   serviceBinding: boolean;
+  eventSourceEnabled: boolean;
```

Comment:
@jeff-phillips-18 do we not have a better way to track feature enablement in topology?

Reply:
Enablement is in State so barring passing it thru all the layers to get from a component into a util there is none. Adding it to the graph data is probably not the best way to store it (things won't get updated on state changes), but that goes for the other stored items that are there now :/. Likely we should remove them all and pass the args around. I'd be OK with a follow-up issue to resolve this.

Reply:
Raised a bug to track https://issues.redhat.com/browse/ODC-3506